loafwallet/AlertFailureView.swift

```diff
@@ -1,60 +1,50 @@
-//
-//  AlertFailureView.swift
-//  loafwallet
-//
-//  Created by Kerry Washington on 1/29/21.
-//  Copyright © 2021 Litecoin Foundation. All rights reserved.
-//
-
 import SwiftUI
 
 struct AlertFailureView: View {
-    
-    let alertFailureType: AlertFailureType
-    
-    let errorMessage: String
-    
-    init(alertFailureType: AlertFailureType, errorMessage: String) {
-        self.alertFailureType = alertFailureType
-        self.errorMessage = errorMessage
-    }
-    
-    var body: some View {
-        VStack {
-            Text(alertFailureType.header)
```

Comment:
The norm for indentation in my Xcode preferences has been 4 spaces...here we are going to 8. Is this intentional or a SwiftFormat default?  It looks cleaner just need to know to change my settings @Iferencak

Reply:
@kcw-grunt It looks a lot cleaner in my xCode, not sure why github presented it this way...

![image](https://user-images.githubusercontent.com/19799406/204549556-6af3c460-b0f1-4778-b7fb-e18216903491.png)

Reply:
But I see now if you try and higlight the empty space it just shows as one tab. So it should be ok.